db/migrate/20230817061119_add_user_logins.rb

```diff
@@ -0,0 +1,31 @@
+class AddUserLogins < ActiveRecord::Migration[7.0]
+  def up
+    create_table :user_logins do |t|
+      t.references :user
+      t.datetime :login_at, null: false
+    end
+
+    execute <<-SQL
+      INSERT INTO user_logins (user_id, login_at)
+      SELECT id, last_login_at FROM users;
+    SQL
```

Comment:
I know, it is not the best solution put raw SQL in the migration to migrate data from one table to another. But it works in this small project.
For real projects which have a big codebase and database, in such cases, I would use a data-migrate gem or rake task to create separate migration for data. The development should look like this: 
1. Create migration for the new table, create migration for data, change code to save data into the new table
2. Test, release to prod, and check that all work well
3. Create migration to remove the old column
4. Test, release changes